content/news/buzz/2023q3-roadmap/contents.lr

```diff
@@ -0,0 +1,38 @@
+title: 2023Q3 Roadmap
+---
+author: Russell Keith-Magee
+---
+body:
+
+2023 is halfway done, and BeeWare continues to make slow but steady progress. As always, this roadmap should be read as a guide to what we aim to focus on over the coming quarter, rather than a hard commitment of features that will be made available on a specific deadline.
```

Comment:
Let's not put ourselves down too much. 😄 

```suggestion
2023 is halfway done, and BeeWare continues to make steady progress. As always, this roadmap should be read as a guide to what we aim to focus on over the coming quarter, rather than a hard commitment of features that will be made available on a specific deadline.
```